src/jbake/content/blog/2017/20170401-second-meetup.md

```diff
@@ -0,0 +1,21 @@
+title=O segundo meetup do VigoJUG
+date=2017-04-01
+type=post
+tags=blog
+status=published
+~~~~~~
+
+
+Despois do ben que o pasamos na primeira meetup (polo menos os organizadores), xa nos tarda organizar a segunda, así que vela ahi vai:
+
+Empezará [Alejandro Moratinos](https://www.linkedin.com/in/alejandro-moratinos-garc%C3%ADa-37395b2a/) de [NetCentric](http://www.netcentric.biz/) cunha charla de 30 minutos titulada: **"Introducción de Cassandra con Java"** na que se fará unha pequena introdución o mundo NoSQL centrado especialmente na base de datos Open Source chamada Cassandra. A continuación farase algún exemplo para ver como utilizala e conectarse desde aplicativos Java, resaltando as boas prácticas e erros máis comúns.
+
+A continuación, virá [Saúl Molinero](https://twitter.com/_saulmm) de [Potsy](https://www.mypopsy.com/) e un dos organizadores do [GDGVigo](http://gdgvigo.com), con outra charla/taller de 30 minutos titulada **"Reactive programming con RxJava"**. Tratase dunha librería para crear programas asíncronos de maneira reactiva. A charla ten utilidade para os linguaxes que se executan na JVM pero tamén para os outros, xa que hai librerías para moitos deles que baséanse nas Reactive Extensions. Tamén se verá a boa cohesión que hai con determinados aspectos de Java 8.
```

Comment:
Feito!

Outra cousa, cambié un pouco o texto, para que fora máis parecido a parte de Alejandro, ¿paréceche ben?